pyro/advection_ppm/problems/_smooth.defaults

```diff
@@ -0,0 +1,2 @@
+[smooth]
```

Comment:
these should all be done as soft links -- see how `advection_rk` links in `problems/` -- this way you don't duplicate code

Reply:
I wonder what do you mean by soft links?